### PR TITLE
Deprecation of support for WS2016 - Changelog

### PR DESCRIPTION
**Pull request by radhikaj**
Signed-off-by: Radhika Jandhyala <radhikaj@microsoft.com>
See related issue #3539

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+### Deprecated
+
+The Open Enclave SDK will be dropping support for WS2016 after Dec 2020. Developers and partners using WS2016 will need to move to using WS2019 by then.
```
Comment (CodeMonkeyLeet) [inline CHANGELOG.md:15]:
```suggestion
- The Open Enclave SDK will be dropping support for WS2016 after Dec 2020.
Developers and partners using WS2016 will need to move to using WS2019 by then.
https://github.com/openenclave/openenclave/issues/3539 tracks this.
```

**Review by CodeMonkeyLeet — APPROVED** [review]
LGTM, see editorial comment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+### Deprecated
+
+- The Open Enclave SDK will be dropping support for WS2016 after Dec 2020.
+Developers and partners using WS2016 will need to move to using WS2019 by then.
+https://github.com/openenclave/openenclave/issues/3539 tracks this.
+https://github.com/openenclave/openenclave/issues/3539 tracks this.
```
Comment (dthaler) [inline CHANGELOG.md:18]: Looks like a duplicated line

Comment (radhikaj) [reply CHANGELOG.md]: Fixed this. Thanks

**Review by mingweishih — APPROVED** [review]
LGTM

**Pull request merged**